lib/Twig/Extension/Escaper.php

```diff
@@ -77,7 +84,7 @@ public function getDefaultStrategy($filename)
     {
         // disable string callables to avoid calling a function named html or js,
         // or any other upcoming escaping strategy
-        if (!is_string($this->defaultStrategy) && is_callable($this->defaultStrategy)) {
```

Comment:
@Tobion I suggest making this change in 1.x.

and we could even validate it in the setter